### DISCUSSION TITLE
What does the new free tier of GitHub Copilot (2024) include?

### DISCUSSION THREAD

**Discussion by AndresXLP** [community/community#147607] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

That is, what languages ​​and frameworks does the free version of GitHub Copilot allow you to use?

**Comment by davevad93** [comment, marked as answer]
Hi @AndresXLP ,

Check the [FAQs on GitHub Docs](https://docs.github.com/en/copilot/managing-copilot/managing-copilot-as-an-individual-subscriber/about-github-copilot-free) about GitHub Copilot Free, it's all explained in detail there.
